src/scala/com/github/johnynek/bazel_deps/DepsModel.scala

```diff
@@ -141,7 +141,10 @@ case class ArtifactOrProject(asString: String) {
     else Nil
 
   def toArtifact(sp: Subproject): ArtifactOrProject =
-    ArtifactOrProject(s"$asString-${sp.asString}")
+    sp.asString match {
+      case "" => this
+      case _ => ArtifactOrProject(s"$asString-${sp.asString}")
```

Comment:
@johnynek Sure, can do.  Is there a more idiomatic way than `val str = sp.asString; str match ... `?

For my own curiosity: `sp.asString` is a public `val` of the Subproject case class -- is there a noticeable overhead to calling it twice, or is it more for readability you'd prefer to see it captured.